filebeat/tests/system/test_reload_modules.py

```diff
@@ -170,6 +170,7 @@ def test_load_configs(self):
         self.render_config_template(
             reload_path=self.working_dir + "/configs/*.yml",
             reload_type="modules",
+            reload=True,
```

Comment:
This is a change in the test behaviour. It's possible to specify `filebeat.config.inputs.path: ...` which loads the config only once.